src/test/java/io/vertx/ext/dropwizard/MetricsTest.java

```diff
@@ -260,8 +264,12 @@ public void testHttpMethodAndUriMetrics() throws Exception {
     vertx.setTimer(100, id -> {
       // Gather metrics
       JsonObject metrics = metricsService.getMetricsSnapshot(server);
-      assertCount(metrics.getJsonObject("get-requests"), 1L);
+      assertCount(metrics.getJsonObject("get-requests"), 4L);
       assertCount(metrics.getJsonObject("get-requests./get"), 1L);
+      assertCount(metrics.getJsonObject("get-requests.users"), 3L);
```

Comment:
Please correct me if I'm wrong about how the "assertCount" works. As I understood it counts not the amount of metrics but rather asserts that the value of the "count" field for some particular metric equals to some number. 
So basically the test proofs that after setting up the identifier "users" and making 3 requests with different path parameters we still have only one entry in metrics registry with the name "get-requests.users" the count of requests for which is 3.